### PR TITLE
Add DSTU 2 documentation for Encounter.

### PR DESCRIPTION
**Pull request by dayouJ**

### CODE REVIEW

File: lib/dstu2_resources.rb [hunk 1]
```diff
@@ -1852,6 +1852,138 @@ module Resources
...
+        ],
+        "entry": [
+          {
+            "fullUrl": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Encounter/2457918",
+            "resource": {
+              "resourceType": "Encounter",
+              "id": "2457918",
+              "text": {
+                "status": "generated",
+                "div": "<div><p><b>Encounter</b></p><p><b>Location</b>: BW Rheumatology</p><p><b>Type</b>: outpatient</p><p><b>Status</b>: finished</p><p><b>Period Start Date</b>: 2015-03-12T15:30:00.000Z</p><p><b>Period End Date</b>: 2015-03-12T16:15:00.000Z</p><p><b>Reason For Visit</b>: rheumatoid arthritis</p><p><b>Attending Physician</b>: Venkat, Savitha</p></div>"
```
Comment (jfrhong) [inline lib/dstu2_resources.rb]: The `<` need to be escaped as `&lt;`, or else it does weird things.

File: lib/dstu2_resources.rb [hunk 2]
```diff
@@ -1852,6 +1852,138 @@ module Resources
...
+                          "code": "ATND",
+                          "display": "attender"
+                        }
+                      ],
+                      "text": "attender"
+                    }
+                  ],
+                  "individual": {
+                    "reference": "Practitioner/4340404",
+                    "display": "Venkat, Savitha"
```
Comment (jfrhong) [inline lib/dstu2_resources.rb]: Switch names to fake names or remove

File: content/dstu2/encounter.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## Retrieve by id
+
+List an individual Encounter by its id:
+
+    GET /Encounter/:id
+
+### Response
+
+<%= headers 200 %>
+<%= json(:dstu2_encounter) %>
```
Comment (jfrhong) [inline content/dstu2/encounter.md]: I don't see a ruby object for this.

File: lib/dstu2_resources.rb [hunk 4]
```diff
@@ -1852,6 +1852,138 @@ module Resources
...
+    DSTU2_ENCOUNTER_BUNDLE ||=
+      {
+        "resourceType": "Bundle",
+        "id": "d41490ce-aece-4de5-ae7b-d4e1c671dd94",
+        "type": "searchset",
+        "total": 2,
+        "link": [
+          {
+            "relation": "self",
+            "url": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Encounter?patient=2744010"
```
Comment (davischristi) [inline lib/dstu2_resources.rb]: Tweak your ids to be fake and not point to actual real data.

File: lib/dstu2_resources.rb (hunk 1 above)
Comment (dayouJ) [inline lib/dstu2_resources.rb]: Fixed here: https://github.com/cerner/fhir.cerner.com/pull/47

File: lib/dstu2_resources.rb (hunk 2 above)
Comment (dayouJ) [inline lib/dstu2_resources.rb]: fixed here: https://github.com/cerner/fhir.cerner.com/pull/47

File: content/dstu2/encounter.md (hunk 3 above)
Comment (dayouJ) [inline content/dstu2/encounter.md]: fixed here: https://github.com/cerner/fhir.cerner.com/pull/47

File: lib/dstu2_resources.rb (hunk 4 above)
Comment (dayouJ) [inline lib/dstu2_resources.rb]: fixed.

**Comment by dayouJ** [comment]
When i did the rebase i accidentally squashed my commit into Naveen's preview commit. Closing this pull request, have a new branch here:https://github.com/cerner/fhir.cerner.com/pull/47

**Pull request closed without merging**